python/tk_softimage/menu_generation.py

```diff
@@ -103,7 +103,9 @@ def _add_context_menu(self):
         ctx_name = str(ctx)
 
         # create the sub menu object
-        ctx_menu = self._menu_handle.AddSubMenu(ctx_name)
+        # the label expects a unicode object so we cast it to support when the context may 
+        # contain info with non-ascii character
+        ctx_menu = self._menu_handle.AddSubMenu(ctx_name.decode("utf-8"))
```

Comment:
nice one! In that case, we should probably check that it is a string object and not a unicode prior to conversion, right? The current location of the code is very explicit, first it does `ctx_name = str(ctx)` and then does a `decode()` . I think we we move the decode into the general `AddSubMenu` method, we don't know what sort of string object is passed so maybe makes sense to check if it is a unicode object and only convert then!